src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/LoggingSelectionInfo.cs

```diff
@@ -0,0 +1,156 @@
+		: base(collection, isGrouped, itemsPath)
+	{
+	}
+
+	public void SelectRange(ItemIndexRange itemIndexRange)
+	{
+		MethodLog.Add($"SelectRange({itemIndexRange.FirstIndex}, {itemIndexRange.Length})");
+	}
+
+	public void DeselectRange(ItemIndexRange itemIndexRange)
+	{
+		MethodLog.Add($"DeselectRange({itemIndexRange.FirstIndex}, {itemIndexRange.Length})");
+	}
+
+	public bool IsSelected(int index)
+	{
+		MethodLog.Add($"IsSelected({index})");
+		return false;
+	}
+
+	public IReadOnlyList<ItemIndexRange> GetSelectedRanges()
+	{
+		MethodLog.Add("GetSelectedRanges()");
+		return new List<ItemIndexRange>();
+	}
+
+	public void LogAndMoveCurrentToPosition(int index)
+	{
+		MethodLog.Add($"MoveCurrentToPosition({index})");
+		base.MoveCurrentToPosition(index);
+	}
+
+	public void LogAndMoveCurrentTo(object item)
+	{
+		MethodLog.Add($"MoveCurrentTo({item})");
+		base.MoveCurrentTo(item);
+	}
+
+	public void LogAndMoveCurrentToFirst()
+	{
+		MethodLog.Add("MoveCurrentToFirst()");
+		base.MoveCurrentToFirst();
+	}
+
+	public void LogAndMoveCurrentToLast()
+	{
+		MethodLog.Add("MoveCurrentToLast()");
+		base.MoveCurrentToLast();
+	}
+
+	public void LogAndMoveCurrentToNext()
+	{
+		MethodLog.Add("MoveCurrentToNext()");
+		base.MoveCurrentToNext();
+	}
+
+	public void LogAndMoveCurrentToPrevious()
+	{
+		MethodLog.Add("MoveCurrentToPrevious()");
+		base.MoveCurrentToPrevious();
+	}
+
+	private bool IsUsingISelectionInfo => true;
+
+	public void SafeMoveCurrentTo(object item)
+	{
+		if (IsUsingISelectionInfo)
+		{
+			MethodLog.Add("MoveCurrentTo() - Skipped due to ISelectionInfo usage");
+		}
+		else
+		{
+			LogAndMoveCurrentTo(item);
+		}
+	}
+
+	public void SafeMoveCurrentToPosition(int index)
+	{
+		if (index == -1)
+		{
+			MethodLog.Add("MoveCurrentToPosition() - Skipped due to invalid index (-1)");
+			return;
+		}
+		if (IsUsingISelectionInfo)
+		{
+			MethodLog.Add("MoveCurrentToPosition() - Skipped due to ISelectionInfo usage");
+		}
+		else
+		{
+			LogAndMoveCurrentToPosition(index);
+		}
+	}
+
+	public void SafeMoveCurrentToFirst()
+	{
+		if (IsUsingISelectionInfo)
+		{
+			MethodLog.Add("MoveCurrentToFirst() - Skipped due to ISelectionInfo usage");
+		}
+		else
+		{
+			LogAndMoveCurrentToFirst();
+		}
+	}
+
+	public void SafeMoveCurrentToLast()
+	{
+		if (IsUsingISelectionInfo)
+		{
+			MethodLog.Add("MoveCurrentToLast() - Skipped due to ISelectionInfo usage");
+		}
+		else
+		{
+			LogAndMoveCurrentToLast();
+		}
+	}
+
+	public void SafeMoveCurrentToNext()
+	{
+		if (IsUsingISelectionInfo)
+		{
+			MethodLog.Add("MoveCurrentToNext() - Skipped due to ISelectionInfo usage");
+		}
+		else
+		{
+			LogAndMoveCurrentToNext();
+		}
+	}
+
+	public void SafeMoveCurrentToPrevious()
+	{
+		if (IsUsingISelectionInfo)
+		{
+			MethodLog.Add("MoveCurrentToPrevious() - Skipped due to ISelectionInfo usage");
+		}
+		else
+		{
+			LogAndMoveCurrentToPrevious();
+		}
+	}
+}
```

Comment:
None of all this code does anything ... copilot/chat GPT are only assistants, you have to validate what they are suggesting ^^ You can simply tests that by commenting the fix you did in the `ComboBox` and check if your test is passing or not.

Again the idea is to validate that, if the a class that implements both the `ISelectionInfo` and `ICollectionView` interfaces, only the `ISelectionInfo` is being used by the `ComboBox`.

For this, please restore the initial approach and make your class to directly implement the `ICollectionView`. The issues you were getting on the CI was only because you didn't implemented all the members of the interface (it's possible that interface is not the same on all platforms, if so, you will have to add members of ALL platforms).

You should be able to simply copy paste this https://github.com/unoplatform/uno.extensions/blob/main/src/Uno.Extensions.Reactive.UI/Presentation/Bindings/Collections/Views/BasicView.cs, remove every parameters of the constructor, and then depending on which field is being used do the following:

| Field used in the `BasicView.cs` | Your implementation |
| -- | -- |
| `_collectionChanged` | Ignore |
| `_collection` | Defer to a private `List<object>` |
| `_edition` | Ignore |
| `_selection` |  Log in the `MethodLog` |
| `_pagination` |  Ignore |